pkg/server/plugin/datastore/sql/sqlite.go

```diff
@@ -0,0 +1,26 @@
+package sql
+
+import (
+	"github.com/jinzhu/gorm"
+	_ "github.com/jinzhu/gorm/dialects/sqlite"
+)
+
+type sqlite struct{}
+
+func (s sqlite) connect(connectionString string) (*gorm.DB, error) {
+
```

Comment:
was this newline intentional? and the one at the end?

Reply:
Was not intentional and has been removed.